docs/01_setup.md

```diff
@@ -145,13 +152,13 @@ cd apps/subgraph
 }
 ```
 
-4. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
-   - Create an `.env` file, and run the follow command in the console `export $(xargs < .env)`
-5. Run `graph auth --studio {key}`. You can find the key in subgraph studio dashboard.
-6. Run `pnpm run deploy` to deploy subgraph
-
-> [!IMPORTANT]
-> The `pnpm run deploy` command call `maci-subgraph` as the subgraph name as default, but if you named your subgraph differently (e.g. maci-graph, my-graph, etc.), please change the command to `graph deploy --node https://api.studio.thegraph.com/deploy/ :your_subgraph_name`
+4. If you have not yet installed the graph CLI in your computer globally do so by running `pnpm i -g @graphprotocol/graph-cli`
+5. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
+   - Create an `.env` file, and run the following command in the console `export $(xargs < .env)`
+6. Copy the auth command from the Subgraph Studio graph details in section "Auth and Deploy", paste it into the terminal and execute it. I should look something like this `graph auth --studio {key}`.
+7. Change `package.json` deploy script to graph deploy --node https://api.studio.thegraph.com/deploy/<name> where <name> is the name that you gave to the subgraph you created.
```

Comment:
The command is 
```bash
graph deploy --node https://api.studio.thegraph.com/deploy/ <name>
```
there is a space between `{blah blah blah...}/deploy/` and `<name>`